Multisig/Logic/Models/Safe.swift

```diff
@@ -62,6 +63,23 @@ extension Safe: Identifiable {
 
         CoreDataStack.shared.saveContext()
     }
+    
+    static func edit(address: String, name: String, selected: Bool = true) {
+        let context = CoreDataStack.shared.viewContext
+
+        let fr = Safe.fetchRequest().by(address: address)
+        
+        guard let result = try? context.fetch(fr) else { return }
```

Comment:
you could do just `guard let result = try? context.fetch(fr).first else  { return } `